StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentOptionsViewController.swift

```diff
@@ -30,6 +30,13 @@ protocol SavedPaymentOptionsViewControllerDelegate: AnyObject {
 /// For internal SDK use only
 @objc(STP_Internal_SavedPaymentOptionsViewController)
 class SavedPaymentOptionsViewController: UIViewController {
+    enum SavedPaymentOptionsViewControllerError: Error {
```

Comment:
Same comment as another PR - should this just be `enum Error: Swift.Error` since it's already in the Saved...VC namespace?